understanding/accessible-authentication-minimum.html

```diff
@@ -336,97 +252,69 @@ <h3>Object Recognition</h3>
                   </ul>
                   
                   
-                  <p>None of these systems are 100% effective. However, they may reduce the likelihood
-                     of a CAPTCHA being displayed.
+                  <p>これらのシステムはいずれも 100％有効ではない。しかし、CAPTCHA が表示される可能性を減らすことはできる。
                   </p>
                   
                </section>
                <section id="personal-content">
                   
-                  <h3>Personal Content</h3>
+                  <h3>個人特有のコンテンツ</h3>
                   
-                  <p>Personal content is sometimes used as a second factor for authentication. For example,
-                     as part of account creation the user would upload a picture, and when logging in they
-                     would be asked to select that picture from several possible alternatives. Care must
-                     be taken to provide adequate security in this case, since non-legitimate users might
-                     be able to guess the correct personal content when presented with a choice.
+                  <p>個人特有のコンテンツは、認証の第 2 要素として使用されることがある。例えば、アカウント作成の一環として利用者が写真をアップロードし、ログイン時にいくつかの可能な選択肢からその写真を選択するように求められる。この場合、非正規の利用者が選択肢を提示されたときに正しい個人特有なコンテンツを推測できる可能性があるため、適切なセキュリティを提供するように注意しなければならない。
                   </p>
                   
                   
-                  <p>Text-based personal content does not qualify for this exception as it relies on recall
-                     (rather than recognition), and transcription (rather than selecting an item). Whilst
-                     picture-based personal content will still be a barrier for some people, text based
-                     versions tend to be a much larger barrier.
+                  <p>テキストベースの個人特有のコンテンツは、(認識ではなく) 記憶及び (項目の選択ではなく) 転記に依存しているため、この例外の対象にはならない。写真ベースの個人特有のコンテンツが依然として障壁となる人もいるが、テキストベースのバージョンははるかに大きな障壁となる傾向がある。
```

Comment:
> (認識ではなく) 記憶
> // 原文では「recall (rather than recognition)」)

は、Jacob Nielsen の [10 Usability Heuristics](https://www.nngroup.com/articles/ten-usability-heuristics/) のひとつ「Recognition Rather than Recall」に由来していて、ここで言う recognition は、ニュアンスとしては、「記憶を呼び起こさなくても、見ただけでわかる」という感じなのですが、それでも「認識」と訳すしかなさそうですね。(これといった簡潔な代替案が浮かばず...)

Reply:
これは主査レビューに丸投げしますか…ｗ

Reply:
「テキストベースの個人特有のコンテンツ」というのはいわゆる「秘密の合言葉」みたいなやつですね。写真を選ぶとかではなく言葉を思い出さないといけないのでダメ (例外として認められない) です、という話。

recallとremeberの訳しわけなどが微妙に気にならなくもないですが、基本、記憶/認識で良いように思います。読みづらい感じはするのですが、もしかすると「及び」の位置が微妙だからかもしれません。原文と語順を揃えてこうすると多少読みやすいかも?

```suggestion
                  <p>テキストベースの個人特有のコンテンツは、記憶 (認識ではない) 及び転記 (項目の選択ではない) に依存しているため、この例外の対象にはならない。写真ベースの個人特有のコンテンツが依然として障壁となる人もいるが、テキストベースのバージョンははるかに大きな障壁となる傾向がある。
```